src/radio-button/radio-button.styles.tsx

```diff
@@ -22,15 +22,23 @@ export const Container = styled.div<StyleProps>`
     width: 1.5rem;
     position: relative;
     border-radius: 50%;
-    border: 2px solid ${V2_Color.Accent.Light[2]};
+    border: 2px solid ${Colour["icon-primary-subtlest"]};
 
     ${(props) => {
         if (props.$disabled) {
             return `
-				border: 2px solid ${V2_Color.Neutral[4](props)};
+				border: 2px solid ${Colour["border-disabled"](props)};
```

Comment:
can refactor this to use `css` so you don't have to forward the props to the Colour fn